### PR TITLE
LPS-80375 Create sample to use Analytics Core and Analytics Forms Plugin

### PR DESCRIPTION
**Pull request by allanragec**

### CODE REVIEW

File: Sample/analytics-ios-demo/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+/**
+- Author: Allan Melo
+*/
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+	var window: UIWindow?
+	
+	func applicationDidFinishLaunching(_ application: UIApplication) {
+		try! Analytics.init(analyticsKey: "DEMOAPPSAMPLEIOSALLA_NEW")
```
[the remark this answers is not in the document]
Comment (allanragec) [reply Sample/analytics-ios-demo/AppDelegate.swift]: Maybe can we change to "configure" ?? like Firebase... i don't know if the name is bad... we chose this name explicitly. What do you think ??

File: Plugins/liferay-analytics-forms/Source/Bind/Forms.swift [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+					return
+				}
+				
+				let currentDate = Date()
+				let focusDuration =
+					Int(currentDate.timeIntervalSince1970 - focusedDate.timeIntervalSince1970)
+				
+				self.fieldBlurred(fieldAttributes: fieldAttributes, focusDuration: focusDuration)
+			})
+			.disposed(by: disposeBag)
```
[the remark this answers is not in the document]
Comment (allanragec) [reply Plugins/liferay-analytics-forms/Source/Bind/Forms.swift]: much cleaner, thanks for the suggestion!

File: Plugins/liferay-analytics-forms/Source/Bind/Forms.swift [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+		
+		let observableBlur = field
+			.rx
+			.controlEvent(UIControlEvents.editingDidEnd)
+			.asObservable()
+		
+		trackField(
+			observableFocus: observableFocus, observableBlur: observableBlur,
+			disposeBag: field.rx.disposeBag, fieldAttributes: fieldAttributes)
+	}
```
[the remark this answers is not in the document]
Comment (allanragec) [reply Plugins/liferay-analytics-forms/Source/Bind/Forms.swift]: nice 👍

File: Sample/analytics-ios-demo/AppDelegate.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (victorg1991) [reply Sample/analytics-ios-demo/AppDelegate.swift]: Ohhh, I thought It was a constructor, but yeah, I think this way is misleading. Configure looks nice to me 👍

**Comment by igormatos** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**